python/langsmith/client.py

```diff
@@ -4664,6 +4680,543 @@ def _evaluate_strings(
             **kwargs,
         )
 
+    def _current_tenant_is_owner(self, owner: str) -> bool:
+        """Check if the current workspace has the same handle as owner.
+
+        Args:
+            owner (str): The owner to check against.
+
+        Returns:
+            bool: True if the current tenant is the owner, False otherwise.
+        """
+        settings = self._get_settings()
+        return owner == "-" or settings["tenant_handle"] == owner
```

Comment:
Would be kinda nice to TypedDict the settings at some opint

Reply:
added